pkgs/misc/wiki-tui/default.nix

```diff
@@ -1,19 +1,25 @@
-{ lib, rustPlatform, fetchCrate, ncurses, openssl, pkg-config }:
+{ lib, rustPlatform, fetchFromGitHub, ncurses, openssl, pkg-config, nix-update-script }:
 
 rustPlatform.buildRustPackage rec {
   pname = "wiki-tui";
-  version = "0.3.4";
+  version = "0.4.1";
 
-  src = fetchCrate {
-    inherit pname version;
-    sha256 = "sha256-H+C1AbZ2zUhw6TNlSPaNaNuY5iNf39JW4q2g6uolevM=";
+  src = fetchFromGitHub {
+    owner = "Builditluc";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-aCfYb5UvlWbBvBlwNjVIbOJlEV2b3+FD0qKO+9h5Nos=";
   };
 
   buildInputs = [ ncurses openssl ];
 
   nativeBuildInputs = [ pkg-config ];
 
-  cargoSha256 = "sha256-PIt592nGtNEREQMukqRl/6KSJ/P32fWucHEMyOOc7BA=";
+  cargoSha256 = "sha256-dD8iC/b6KwElmq0RNdnHHC/jMWh7Q0/kMM+/zityHqQ=";
+
+  passthru.updateScript = nix-update-script {
+    attrPath = pname;
+  };
```

Comment:
This can also be done with nix-update which does not require any changes to the package.

Reply:
This is less work for me, I'd rather like to have the package be updated automatically